pyEpiabm/pyEpiabm/sweep/assign_household_ages_sweep.py

```diff
@@ -0,0 +1,389 @@
+
+        r = random.random()
+
+        # case where an elderly person lives alone
+        if r < self.age_params["one_pers_house_prob_old"]:
+            while True:
+                person.set_random_age()
+                break_ratio = ((person.age
+                                - self.age_params["no_child_pers_age"] + 1)
+                               / (self.age_params["old_pers_age"]
+                               - self.age_params["no_child_pers_age"]
+                               + 1))
+                if ((person.age >= self.age_params["no_child_pers_age"]) and
+                   (random.random() <= break_ratio)):
+                    break
+
+        # case where a young and single adult lives alone
+        elif ((self.age_params["one_pers_house_prob_young"] > 0) and
+              (r < (self.age_params["one_pers_house_prob_young"]
+               / (1 - self.age_params["one_pers_house_prob_old"])))):
+            while True:
+                person.set_random_age()
+                break_ratio = (
+                    1 - self.age_params["young_and_single_slope"]
+                    * ((person.age - self.age_params["min_adult_age"])
+                        / (self.age_params["young_and_single"]
+                           - self.age_params["min_adult_age"])))
+                if ((person.age <= self.age_params["young_and_single"]) and
+                   (person.age >= self.age_params["min_adult_age"]) and
+                   (random.random() <= break_ratio)):
+                    break
+
+        # case where it is just one adult living alone of any age
+        # older than minimum adult age
+        else:
+            while True:
+                person.set_random_age()
+                if person.age >= self.age_params["min_adult_age"]:
+                    break
+
+    def two_person_household_ages(self, people):
+        """Method that assigns ages to the people
+        in a two person household. A random number is first drawn
+        that decides which set of conditions the people's ages
+        should satisfy, e.g. if they are both old or it is one parent
+        and one child etc. The people's ages are then repeatedly set
+        until they are realistic e.g their ages are compatible
+        if they are a couple.
+
+        Parameters
+        ----------
+        People : list
+            List of two instances of Person class
+
+        """
+
+        r = random.random()
+        person1 = people[0]
+        person2 = people[1]
+        assert len(people) == 2,\
+               'Only a list of two people should be passed to this method'
+
+        # case where two elderly people live alone
+        if r < self.age_params["two_pers_house_prob_old"]:
+            while True:
+                person1.set_random_age()
+                break_ratio = ((person1.age
+                               - self.age_params["no_child_pers_age"] + 1)
+                               / (self.age_params["old_pers_age"]
```

Comment:
I'll be honest I am not finding these names easy to understand - I think this will be easier to review when the wiki is complete!